services/src/handlers/datasets.rs

```diff
@@ -169,29 +169,34 @@ pub async fn list_datasets_handler<C: ApplicationContext>(
         (status = 400, description = "Bad request", body = ErrorResponse, examples(
             ("Referenced an unknown dataset" = (value = json!({
                 "error": "CannotLoadDataset",
-                "message": "CannotLoadDataset: UnknownDatasetId"
+                "message": "CannotLoadDataset: UnknownDatasetName"
             })))
         )),
         (status = 401, response = crate::api::model::responses::UnauthorizedUserResponse)
     ),
     params(
-        ("dataset" = DatasetId, description = "Dataset id")
+        ("dataset" = DatasetName, description = "Dataset Name")
     ),
     security(
         ("session_token" = [])
     )
 )]
 pub async fn get_dataset_handler<C: ApplicationContext>(
-    dataset: web::Path<DatasetId>,
+    dataset: web::Path<DatasetName>,
```

Comment:
I can't comment the right line, but the doc comment still says "internal id" which is no longer true

Reply:
[927b174](https://github.com/geo-engine/geoengine/pull/796/commits/927b174498f2a3bb9df7199d06f082ada6cdb218)